src/queues/sqs/notification/test.rs

```diff
@@ -243,3 +243,47 @@ fn serialize_complaint_feedback_type() {
     let json = serde_json::to_string(&ComplaintFeedbackType::Virus).expect("JSON error");
     assert_eq!(json, "\"virus\"");
 }
+
+#[test]
+fn deserialize_notification() {
+    let json = include_str!("test_notifications.json");
+    let notifications: Vec<Notification> = serde_json::from_str(json).expect("JSON error");
+
+    assert_eq!(notifications.len(), 3);
+
+    assert_eq!(notifications[0].notification_type, NotificationType::Bounce);
+    assert!(notifications[0].bounce.is_some());
+    assert!(notifications[0].complaint.is_none());
+    assert!(notifications[0].delivery.is_none());
+    let bounce = match notifications[0].bounce {
+        Some(ref record) => record,
+        _ => panic!("Bounce record should exist."),
+    };
+    assert_eq!(bounce.bounced_recipients.len(), 3);
```

Comment:
Hey @brizental, one last thing we should assert is that the email addresses in here match the test data. Then we're golden!

Reply:
Done!